test/passing/infix_bind.ml

```diff
@@ -0,0 +1,115 @@
+[@@@ocamlformat "break-infix-before-func=true"]
+
+;;
+f x
+>>= fun y ->
+g y
+>>= fun () ->
+f x >>= fun y -> g y >>= fun () -> f x >>= fun y -> g y >>= fun () -> y ()
+
+;;
+f x
+>>= function
+| A -> (
+    g y
+    >>= fun () ->
+    f x
+    >>= fun y ->
+    g y
+    >>= function x -> ( f x >>= fun y -> g y >>= function _ -> y () ) )
+
+[@@@ocamlformat "break-infix-before-func=false"]
+
+;;
+f x >>= fun y ->
+g y >>= fun () ->
+f x >>= fun y -> g y >>= fun () -> f x >>= fun y -> g y >>= fun () -> y ()
+
+;;
+f x >>= function
+| A -> (
+    g y >>= fun () ->
+    f x >>= fun y ->
+    g y >>= function
+    | x -> ( f x >>= fun y -> g y >>= function _ -> y () ) )
+
+;;
+eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee |> fun x -> x
+
+;;
+eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
+  |> fun xxxxxx xxxxxxxxxx xxxxxxxx xxxxxxxx -> x
```

Comment:
Is the indentation expected? `ocp-indent` will format this as:

```ocaml
eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
|> fun xxxxxx xxxxxxxxxx xxxxxxxx xxxxxxxx -> x
```

Reply:
Not expected to me. It would be good if with `break-infix-before-func=false`, when breaks are needed, the formatting is the same as with `break-infix-before-func=true` wrt indentation, etc.